submissions/franqly/franqly.md

```diff
@@ -0,0 +1,99 @@
+
+{ twitch embed
+    <iframe src="https://clips.twitch.tv/embed?clip=GeniusStrongPeachMVGame-FoTU1sHX3_UzpcSk&parent=localhost" frameborder="0" allowfullscreen="true" scrolling="no" height="378" width="620"></iframe>
+    caption = "Legendary speedrunner [Nefrums](https://www.twitch.tv/nefrums/) makes a tiny mistake that clogs a belt and ultimately cost him the run. Did you catch it?"
+}
+
+## Who is Franqly
+{*conor please change code blocks into quotes. I have them as code to make it easier to read for now*}
+
+Now, who is Franqly? The speedrunning scene was dominated for a long time by names like AntiElitz, Nefrums, and Phoenix27833 - to name only a few. However, Franqly soon came around, and through practice and dedication he became one of the best. I reached out to him to ask him a few questions about his time in Factorio and how he managed to reach the top so fast after only speedrunning `The Legend of Zelda: Ocarina of Time` a few times. He was very gracious and gave us a glimpse into his Factorio career.
+
+*Is there someone specific that got you into Factorio, and how much time have you spent in Factorio?*
+
+`No one specific.  I can't quite remember how i found it, but I probably saw someone playing on twitch.  I thought it looked like s**t tbh, but was interested in the concept.  Then a few months after, i saw it on steam.  It looked much better with .12, and decided to pick it up.  I had about 1500 hours before starting to speedrun. [I am at] about 2200 [hours] now.`
+
+*When did you start speedrunning, and is there someone who inspired you to start? Is there someone now that you see as a type of mentor?*
+
+`I started relatively recently, about August last year.  I had contemplated it in the past, as I had watched Anti's streams off and on for a while.  Nefrums created an introduction to soeedrunning guide, and that gave me the push to try it.  I'd recommend that guide to anyone thinking about trying out factorio speedrunning.  I would consider Nef the most mentor like person in the community for me, but everyone in the discord and on twitch chat chimes in with advice and strategy ideas, and that help me come to speed very quickly.`
+
+*Speedruns tend to look very similar between different players to the untrained eye. What sets your run apart from the others?*
```

Comment:
Something that could be mentioned somewhere around here (if possible with this interview format) is that a bunch of people use the same base plan, e.g. currently 3 out of the top 4 runs use Nefrums' base (Franqly, Nefrums, Pitta). Could also be mentioned around the "hardest part" answers, since it underlines the fact that using other people's designs is considered okay.

When going further down the leaderboard, one could even make a fun statistic like "x amount of people uses y design". If you think that could fit here, I'd be open to helping identify designs.